packages/eas-cli/src/build/runBuildAndSubmit.ts

```diff
@@ -192,6 +195,26 @@ export async function runBuildAndSubmitAsync(
 
   for (const buildProfile of buildProfiles) {
     const platform = toAppPlatform(buildProfile.platform);
+    const { projectId } = await getDynamicPrivateProjectConfigAsync({
+      env: buildProfile.profile.env,
+    });
+    const env = await resolveEnvVarsAsync({
+      flags,
+      buildProfile,
+      graphqlClient,
+      projectId,
+    });
+
+    const buildVariables = { ...env, ...buildProfile.profile.env };
+
+    Log.log(
+      `Loaded "env" configuration for the "${buildProfile.profileName}" profile: ${
+        buildVariables
+          ? Object.keys(buildVariables).join(', ')
+          : 'no environment variables specified'
+      }. ${learnMore('https://docs.expo.dev/build-reference/variables/')}`
+    );
```

Comment:
This message is printed in some other place as well, right? Shouldn't we remove it?